src/requestor/routes/manage.py

```diff
@@ -36,6 +36,40 @@ class CreateRequestInput(BaseModel):
     policy_id: str = None
 
 
+async def grant_or_revoke_arborist_policy(
+    arborist_client, policy_id, username, revoke, status
+):
+    # the access request is approved: grant access
+    if status in config["UPDATE_ACCESS_STATUSES"]:
```

Comment:
i think this `if` belongs in the calling function's logic, and this `grant_or_revoke_arborist_policy` function should start at the following line